services/skus/controllers_test.go

```diff
@@ -311,7 +321,15 @@ func (suite *ControllersTestSuite) setupCreateOrder(skuToken string, token macar
 		suite.Require().NoError(err)
 	}
 
-	issuer, _ := suite.storage.GetIssuer(issuerID)
+	repo := repository.NewIssuer()
+
+	var exp error
+	if credType == timeLimited {
+		exp = model.ErrIssuerNotFound
+	}
```

Comment:
This was previously hidden by suppressing the error (look at the code it changes from). It was not clear and took me ages to understand why. The reason is that we never bother creating/checking issuer for credentials of the `time-limited` type.

Had only this been explained in a comment on top of the line with the suppressed error (as any error suppression should).